### ISSUE TITLE
Building with Carthage on Xcode 10 fails to build ReactiveMapKit

### ISSUE THREAD

**Issue by mluisbrown** [ReactiveCocoa/ReactiveCocoa#3626]
When I try to build using Carthage it fails building ReactiveMapKit.

I have `github "ReactiveCocoa/ReactiveCocoa"` in my `Cartfile`. It's pulling down `8.0.1`.

I do `carthage update --platform iOS` and I get this:

```
*** Building scheme "ReactiveSwift-iOS" in ReactiveSwift.xcworkspace
*** Building scheme "ReactiveCocoa-iOS" in ReactiveCocoa.xcworkspace
*** Building scheme "ReactiveMapKit" in ReactiveCocoa.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa.xcworkspace -scheme ReactiveMapKit -configuration Release -derivedDataPath /Users/michael.brown/Library/Caches/org.carthage.CarthageKit/DerivedData/10.0_10A255/ReactiveCocoa/8.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/qd/qls_wfdd0ws66gthby11gx38xpyg5b/T/ReactiveCocoa SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa)
```

The log file contains many errors:

Many like this:
```
/Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceButton.swift:4:32: error: 'WKInterfaceButton' is only available on iOS application extension 8.2 or newer
extension Reactive where Base: WKInterfaceButton {
                               ^
/Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceButton.swift:4:1: note: add @available attribute to enclosing extension
extension Reactive where Base: WKInterfaceButton {
```

And many like this:
```
/Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:459:41: warning: 'allocate(bytes:alignedTo:)' is deprecated: renamed to 'allocate(byteCount:alignment:)'
                        let buffer = UnsafeMutableRawPointer.allocate(bytes: size, alignedTo: alignment)
                                                             ^
/Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:459:41: note: use 'allocate(byteCount:alignment:)' instead
                        let buffer = UnsafeMutableRawPointer.allocate(bytes: size, alignedTo: alignment)
                                                             ^~~~~~~~ ~~~~~        ~~~~~~~~~
                                                             allocate byteCount    alignment
/Users/michael.brown/Documents/Projects/Memories/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:410:42: warning: 'allocate(bytes:alignedTo:)' is deprecated: renamed to 'allocate(byteCount:alignment:)'
                        let pointer = UnsafeMutableRawPointer.allocate(bytes: MemoryLayout<U>.size,
                                                              ^
```

**Comment by mluisbrown** [comment]
Checked whether doing just `carthage build` with no platform specifier would make a difference. Nope 😞 The thing is I don't even need `ReactiveMapKit` for my app...is there a way to prevent it from building?

**Comment by ikesyo** [comment]
That will be a duplicate of #3619.

**Comment by andersio** [comment]
Resolved by #3625.
